test/view_component/integration_test.rb

```diff
@@ -200,6 +200,7 @@ class IntegrationTest < ActionDispatch::IntegrationTest
     get "/rails/view_components/no_layout/default"
 
     assert_select("div", "hello,world!")
+    refute_includes response.body, "ViewComponent - Test"
```

Comment:
Let's add a new test case for no_layout where we set `default_preview_layout` and make sure that there is still no layout.

Reply:
@jaredcwhite thoughts on @jonspalmer's comment here?

Reply:
@joelhawksley I think I did that in a subsequent commit, but I'll double-check